software/tinyclr/tutorials/networking/intro.md

```diff
@@ -1,9 +1,5 @@
 # Networking
 ---
-It wasn't that long ago that computer networking was expensive, complicated, and used almost solely within organizations. With the widespread adoption of Wi-Fi, Ethernet, and the Internet, networking is now inexpensive, easy, and ubiquitous.
+TinyCLR OS Networking support is still under development. It currently supports WiFi through STMicroelectronics SPWF04Sx secure WiFi module. The implementation can be found in the `GHIElectronics.TinyCLR.Networking` and `GHIElectronics.TinyCLR.Networking.Http` libraries. It provides socket and HTTP APIs that should be familiar to any .NET developer. Take a look at the [.NET docs](https://docs.microsoft.com/en-us/dotnet/api/system.net.sockets.socket) for samples.
```

Comment:
The phrasing implies that the SPWF04Sx is found in the Networking and Http libraries, we should clarify.